### DISCUSSION TITLE
Data-Importer and Transactions

### DISCUSSION THREAD

**Discussion by acatenac** [firefly-iii/firefly-iii#5420] [Firefly III]
I noticed from the dropdowns that there are multiple field mapping options ... does that mean that transactions can also be imported.

If so could the export format be used to model an import template

Thanks

**Comment by JC5** [comment]
I'm not sure exactly what the question is. But yes, you can import transactions from Firefly III itself, using this configuration:

https://github.com/firefly-iii/import-configurations/tree/main/firefly-iii
